bitcoinplusplus/layer-2/exploring-p2p-hashrate-markets-v2.md

```diff
@@ -266,18 +292,17 @@ The problem is, as Lisa calls it, we have terabat liquidity issues with mining p
 You're going to exhaust those channel capacities immediately.
```

Comment:
I can't comment directly on the line.
This comment is for the above line "The problem is, as Lisa calls it, we have terabat liquidity issues with mining pool payouts to miners over lightning, Because we're only pushing liquidity in one direction."

It appears there's a slight transcription error regarding the term used by the speaker. Based on the slides, the correct term is "terribad," not "terabat." Although it's a minor detail, ensuring the accuracy of such terms is crucial for maintaining the fidelity of the transcript to what was actually said.